core/deliverservice/client_test.go

```diff
@@ -165,7 +165,7 @@ func testOrderingServiceConnFailure(t *testing.T, bdc blocksDelivererConsumer) {
 		setupInvoked++
 		return nil
 	}
-	backoffStrategy := func(attemptNum int, elapsedTime time.Duration) (time.Duration, bool) {
+	backoffStrategy := func(attemptNum int, elapsedTime time.Duration, yieldLeadership bool) (time.Duration, bool) {
```

Comment:
As all of these changes are being made to adhere to the updated backoff callback signature, I don't think we need them.

I believe that means this file needs no changes.